Microsoft.Toolkit.Win32/Microsoft.Toolkit.Wpf.UI.Controls/WindowsXamlHostBaseExt.cs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel;
+using System.Windows;
+using System.Windows.Data;
+using Microsoft.Toolkit.Win32.UI.XamlHost;
+using Microsoft.Toolkit.Wpf.UI.XamlHost;
+
+namespace Microsoft.Toolkit.Wpf.UI.Controls
+{
+    /// <inheritdoc />
+    public class WindowsXamlHostBaseExt : WindowsXamlHostBase
```

Comment:
As with the WinForms version, can we just fold this into the base class (even as a partial)?